### DISCUSSION TITLE
Xunit tests are NOT running for the .net8 pereview -5 IN .NET MAUI project.

### DISCUSSION THREAD

**Discussion by Naveen1547** [xunit/xunit#2751] [Question] [answered]
![image](https://github.com/xunit/xunit/assets/87467234/c88486d1-a753-4b66-a7eb-2b1ca32f2edd)
Not able to run xunit tests with .net 7 and .net 8 as well.Please help me out on it.

**Comment by bradwilson** [comment, marked as answer]
In general, .NET 7 is supported (and I have no reason to believe .NET 8 won't work, though I don't install preview versions so I cannot verify this):

![image](https://github.com/xunit/xunit/assets/16944/dfa41256-7ad7-4d6a-82d2-192ac193cc09)

MAUI is not officially supported. If the failure is related to MAUI, then I am unable to provide any assistance.
